.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,6 +17,7 @@ Make sure to implement all relevant entries (see section headers to when they ap
 ### For a newly hosted plugin only
 
 - [ ] Add link to resolved HOSTING issue in description above
+- [ ] Ping the `@jenkins-infra/security-hosting` team to determine the need for a security audit
```

Comment:
Are people outside the org able to ping teams? I remember Hacktoberfest PRs etc. where pings just didn't happen.

Reply:
you have to be an org member to ping a team.

This is the wrong place anyway, no one is creating PRs for newly hosted plugins. 

The GitHub action can ping the security team on hosting requests or y'all can just watch issues

Reply:
So nothing to do in this PR, closing.